playbooks/roles/tarsnap/tasks/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
 
-- include: tarsnap.yml tags=tarsnap
-- include: tarsnapper.yml
-  tags:
-    - tarsnap
-  when: "{{ tarsnap_tarsnapper_conf != None}}"
+- include: install.yml tags=tarsnap
+- include: configure.yml tags=tarsnap
```

Comment:
I get this. This was something which I totally missed it. Thanks for pointing it out. :)

Reply:
I have fixed this now. :)

Reply:
@taranjeet Thanks! One more note about the include statements above: Can you put `tags` on a separate line below each include? For example:

```yaml
- include: install.yml
  tags:
    - tarsnap
```

The code below is using that style as well (instead of the `=` style), and not mixing styles will improve readability :)

Reply:
@itsjeyd : To conditionally run tasks from `configure.yml`, it should be `TARSNAP_KEY` rather than `TARSNAP_CACHE`, because `TARSNAP_CACHE` already has a value (=/var/cache/tarsnap). I have updated this.